pkg/text/print.go

```diff
@@ -90,7 +91,26 @@ func PrintInfoValue(key string, values ...string) {
 		delimCount = 2
 	)
 
-	str := fmt.Sprintf(Bold("%-16s: "), key)
+	var (
+		language     = gotext.GetLanguage()
+		keyTextCount = keyLength - delimCount
+	)
+
+	if strings.HasPrefix(language, "zh") {
+		keyRuneCount := utf8.RuneCountInString(key)
+		switch {
+		case len(key) == keyRuneCount:
+		// these keys are all English words, do nothing, use the existing logic
+		case strings.HasSuffix(language, "TW") && key == "AUR 網址":
```

Comment:
This is liable to change and may be broken :frowning:

Reply:
You are right, let me update this.